dace/frontend/python/replacements.py

```diff
@@ -843,7 +852,8 @@ def _representative_num(dtype: Union[dtypes.typeclass, Number]) -> Number:
     elif issubclass(nptype, np.bool_):
         return np.bool_(True)
     elif issubclass(nptype, Integral):
-        return nptype(np.iinfo(nptype).max)
+        # return nptype(np.iinfo(nptype).max)
+        return 0
```

Comment:
shouldn't this be `nptype(0)`?

Reply:
This was a temporary fix for a specific benchmark (fft) that we couldn't make work anyway for other reasons.
The commented-out line is the correct original code, which in combination with SymPy and complex numbers leads to some infinite loop (inside SymPy). I will revert the change and we can investigate the issue later.